src/main/java/minicraft/level/tile/DoorTile.java

```diff
@@ -18,6 +18,10 @@ public class DoorTile extends Tile {
 	protected Material type;
 	private Sprite closedSprite;
 	private Sprite openSprite;
+	public final JSONObject initialDefaultData = new JSONObject();
+	{
+		initialDefaultData.put("closed", 0);

```

Comment:
In this exsample. There would be no problem to make closed a variable in the class.